nix/grpc.nix

```diff
@@ -12,6 +12,8 @@ stdenv.mkDerivation rec {
     sha256 = "19ldbjlnbc287hkaylsigm8w9fai2bjdbfxk6315kl75cq54iprr";
   };
 
+  NIX_CFLAGS_COMPILE = "-Wno-error";
```

Comment:
Newer compilers (both gcc on Linux and Clang on darwin) emit some warnings that break `-Werror` on this ancient version of `gRPC`.